packages/arb-ts/src/lib/bridge.ts

```diff
@@ -288,20 +291,24 @@ export class Bridge {
 
     const maxGas =
       retryableGasArgs.maxGas ||
-      (
-        await nodeInterface.estimateRetryableTicket(
-          expectedL1GatewayAddress,
-          ethers.utils.parseEther('0.05').add(estimateGasCallValue),
-          l2Dest,
-          estimateGasCallValue,
-          maxSubmissionPrice,
-          sender,
-          sender,
-          0,
-          0,
-          depositCalldata
-        )
-      )[0]
+      BridgeHelper.percentIncrease(
+        (
+          await nodeInterface.estimateRetryableTicket(
+            expectedL1GatewayAddress,
+            ethers.utils.parseEther('0.05').add(estimateGasCallValue),
```

Comment:
.05 eth is a pretty large amount, can we make change this to be configurable as well and maybe drop the default if it's safe? We can make it configurable now and drop it separately then

Reply:
Nm just haven't had coffee 😁